internal/acceptance/sql_table_test.go

```diff
@@ -264,3 +268,100 @@ func TestUcAccResourceSqlTable_Liquid(t *testing.T) {
 		}`,
 	})
 }
+
+func constructManagedSqlTableTemplate(columnInfos []catalog.SqlColumnInfo) string {
+	columnsTemplate := ""
+
+	for _, ci := range columnInfos {
+		ciTemplate := fmt.Sprintf(
+			`
+			column {
+				name      = "%s"
+				type      = "%s"
+				nullable  = %t
+				comment   = "%s"
+			}
+			`, ci.Name, ci.Type, ci.Nullable, ci.Comment,
+		)
+		columnsTemplate += ciTemplate
+	}
+
+	return fmt.Sprintf(`
+		resource "databricks_schema" "this" {
+			name         = "{var.STICKY_RANDOM}"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				this      = "that"
+				something = "else"
+			}
+
+			%s
+			comment = "this table is managed by terraform"
+		}`, columnsTemplate)
+}
+
+func TestUcAccResourceSqlTable_RenameColumn(t *testing.T) {
+	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
+		skipf(t)("databricks_sql_table resource not available on GCP")
+	}
+	unityWorkspaceLevel(t, step{
+		Template: constructManagedSqlTableTemplate([]catalog.SqlColumnInfo{{Name: "name", Type: "string", Nullable: true, Comment: "comment"}}),
+	}, step{
+		Template: constructManagedSqlTableTemplate([]catalog.SqlColumnInfo{{Name: "new_name", Type: "string", Nullable: true, Comment: "comment"}}),
+	})
+}
+
+func TestUcAccResourceSqlTable_AddColumnComment(t *testing.T) {
+	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
+		skipf(t)("databricks_sql_table resource not available on GCP")
```

Comment:
is this true?

Reply:
Following the patterns for all other tests for now, we can address them all as a follow up